test/run_test.py

```diff
@@ -207,7 +207,7 @@
     'test_jit_legacy',
     'test_jit_fuser_legacy',
     'test_jit_fuser_te',
```

Comment:
I think `test_jit_fuser_te` also need to be always run.

Reply:
also re-included